presto-main/src/test/java/io/prestosql/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -138,6 +141,14 @@ public static PlanMatchPattern tableScan(String expectedTableName, Map<String, S
         return result.addColumnReferences(expectedTableName, columnReferences);
     }
 
+    public static PlanMatchPattern tableScanWithMatchers(
+            ObjectMatcher<ConnectorTableHandle> expectedTable,
+            TupleDomain<ObjectMatcher<ColumnHandle>> enforcedConstraints,
```

Comment:
`Predicate<TupleDomain<ColumnHandle>> constraintsMatcher`? And similar for columns?

Reply:
replaced ObjectMatcher with Predicates